README.md

```diff
@@ -0,0 +1,15 @@
+# AvalaraTaxShipmentTax Module
+[![Build Status](https://travis-ci.org/spryker-eco/avalara-tax-shipment-tax.svg)](https://travis-ci.org/spryker-eco/avalara-tax-shipment-tax)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.2-8892BF.svg)](https://php.net/)
+
+AvalaraTaxShipmentTax module provides plugins for extending AvalaraTax module functionality with calculating taxes for shipment.
```

Comment:
* AvalaraTaxShipmentTax module provides functionality to calculate taxes for shipment.

Reply:
Resolved.